src/templates/ObservabilityPackDetails.js

```diff
@@ -32,6 +36,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
       packId: pack.id,
     }
   );
+
+  const getPackUrl = () => {
+    const baseUrl =
+      'https://github.com/newrelic/newrelic-observability-packs/tree/main/packs';
+
+    if (!pack.logoUrl) {
+      return baseUrl;
+    }
+
+    /**
+     * logoUrl looks something like: 'https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/couchbase/logo.svg'
+     */
+    const [packName, ..._] = pack.logoUrl.split('/').slice(-2);
+    return join(baseUrl, packName);
```

Comment:
I think the original intention behind `path` was to join filepaths for file resources. We might want to reach for [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) instead. I think it more accurately describes what we're doing:

```js
const base = 'https://github.com/newrelic/newrelic-observability-packs/tree/main/packs';

const name = pack.logoUrl.split('/').slice(-2, -1);
const url = new URL(name, base);

return url.href;
```

Reply:
swapped to url